packages/moonstone/VideoPlayer/VideoPlayer.module.less

```diff
@@ -422,5 +421,13 @@
 				}
 			}
 		}
+
+		.overlay {
+			&.high-contrast-scrim::after {
+				:global(.enact-a11y-high-contrast) & {
```

Comment:
I think this section could be refactored to use the skinning (`.applySkins`) system, rather than its own method.